ckanext/qa/tasks.py

```diff
@@ -356,6 +357,31 @@ def score_by_sniffing_data(archival, resource, score_reasons):
         return (None, None)
     # Analyse the cached file
     filepath = archival.cache_filepath
+
+    # # if filepath does not exist on disk, then lets see if its an upload or a url
+    # # download to temp space, then delete post checking if can't use in memory checking
+    #
+    # # download file to temp use that for rest of call.
+    # # use logic found in xloader for chunk file downloading
+    # # ilepath = temp file
+    # #    return (None, None)
+    # if not os.path.exists(filepath):
+    #     from ckan import model
```

Comment:
No need for the model since we already have a resource dict with the necessary data.